dll/win32/shdocvw/CNSCBand.cpp

```diff
@@ -93,7 +93,8 @@ CNSCBand::~CNSCBand()
 VOID CNSCBand::OnFinalMessage(HWND)
 {
     // The message loop is finished, now we can safely destruct!
-    static_cast<IDeskBand *>(this)->Release();
+    // HACKFIX: Who did this AddRef? Commenting out Release...
```

Comment:
No hint from git blame?

Reply:
My theory is that there is no AddRef so it would not show up in blame. This Release causes a purecall exception that is hidden by the SendMessage exception handler?